libs/ui/src/components/MediaItem/type/ImageMedia.vue

```diff
@@ -5,9 +5,11 @@
       'is-square image': !original,
       'is-detail': isDetail,
     }">
-    <img
+    <NuxtImg
```

Comment:
This is forbidden please use `component` instead and pass it as prop

Reply:
you mean this

```
<component
  :is="isNuxtImage ? NuxtImg : 'img'"
```
?

Reply:
`NuxtImg` isn't available in `libs/ui` unless you migrate the whole project to nuxt
We managed to use `NuxtLink` on button by passing as prop to component.